app-modules/service-management/tests/ServiceRequest/EditServiceRequestTest.php

```diff
@@ -244,3 +248,73 @@
         ->toEqual($request->except('division_id')->toArray())
         ->and($serviceRequest->fresh()->division->id)->toEqual($request['division_id']);
 });
+
+test('send feedback email if service request is closed', function () {
+    Notification::fake();
+
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    $serviceRequest = ServiceRequest::factory()->create();
+
+    actingAs($user)
+        ->get(
+            ServiceRequestResource::getUrl('edit', [
+                'record' => $serviceRequest,
+            ])
+        )->assertForbidden();
+
+    livewire(EditServiceRequest::class, [
+        'record' => $serviceRequest->getRouteKey(),
+    ])
+        ->assertForbidden();
+
+    $user->givePermissionTo('service_request.view-any');
+    $user->givePermissionTo('service_request.*.update');
+
+    actingAs($user)
+        ->get(
+            ServiceRequestResource::getUrl('edit', [
+                'record' => $serviceRequest,
+            ])
+        )->assertSuccessful();
+
+    $request = collect(EditServiceRequestRequestFactory::new([
+        'status_id' => ServiceRequestStatus::factory()->create([
+            'classification' => SystemServiceRequestClassification::Closed,
+        ])->id,
+    ])->create());
+
+    livewire(EditServiceRequest::class, [
+        'record' => $serviceRequest->getRouteKey(),
+    ])
+        ->fillForm($request->toArray())
+        ->call('save')
+        ->assertHasNoFormErrors();
+
+    assertDatabaseHas(
+        ServiceRequest::class,
+        $request->except(
+            [
+                'division_id',
+                'status_id',
+                'priority',
+            ]
+        )->toArray()
+    );
+
+    $serviceRequest->refresh();
+
+    $contact = $serviceRequest->respondent;
+
+    $contact->notify(new SendClosedServiceFeedbackNotification($serviceRequest));
```

Comment:
Hmm, this should be removed here shouldn't it? With this being here you are not actually testing that the edit above actually sent the notification. You are doing so here.